src/__tests__/ceramic_cas.ts

```diff
@@ -0,0 +1,60 @@
+import {buildServicesFromConfig, Services} from "../index";
+import {AnchorStatus, CeramicApi} from "@ceramicnetwork/common"
+
+const registerChangeListener = function (doc: any): Promise<void> {
+    return new Promise(resolve => {
+        doc.on('change', () => {
+            resolve()
+        })
+    })
+}
+
+describe('Ceramic<->CAS integration', () => {
+    jest.setTimeout(1000 * 60 * 5) // 5 minutes
+    let ceramic: CeramicApi
+
+    beforeAll(async () => {
+        const services: Services = await buildServicesFromConfig();
+        ({ceramic} = services);
```

Comment:
Do we need the parentheses here? What does it do?

Reply:
I get a syntax error without it :confused:

Reply:
ohh maybe because of the de-structured assignment